CHANGELOG.rst

```diff
@@ -201,6 +201,10 @@ Changed
   The role has also been included in the ``common.yml`` playbook to ensure that
   the host database is up to date as soon as possible.
 
+- [debops.resources] Changed behaviour of used groups for templating. Now all
+  groups the host is in, will be used to search for template files.
+  Read the documentation about `resources__templates <https://docs.debops.org/en/master/ansible/roles/debops.resources/defaults-detailed.html#resources-templates>`_ for more information on templating with `debops`.
```

Comment:
The documentation section can be referenced internally, you can do it like this:
```rst
Read the documentation about :ref:`resources__ref_templates` for more details.
```